ui/src/components/PollBody/PollBody.jsx

```diff
@@ -18,6 +18,23 @@ const PollBody = () => {
     const [voters, setVoters] = useState([]);
     const [options, setOptions] = useState([{optionName: "", description:""}, {optionName:"", description:""}]);
 
+    const [members, setMembers] = useState([])
+    const [currentUser, setCurrentUser] = useState({})
+
+    useEffect(() => {
+        API.getAllUsers().then((response) => {
+            setMembers(response.data)
+        }).catch((error) => {
+            console.log(error)
```

Comment:
These calls can actually be alerted to the user using Ant Design's Notification. You can import it with `import {notification} from 'antd';` and then use it like so:
```js
notification.open({
  message: "Cannot send request",
  description: "Something bad happened, whoops." 
});
```